doc/api/process.md

```diff
@@ -683,6 +683,22 @@ process.on('warning', (warning) => {
 
 If `warning` is passed as an `Error` object, the `options` argument is ignored.
 
+## process.dlopenFlags
+<!-- YAML
+added: v8.0.0
+-->
+
+* {integer}
+
+The `process.dlopenFlags` property allows to set the flags passed by
+Node.js on `dlopen()` calls. See the [`os.constants.dlopen`][] documentation.
+
+```js
+process.dlopenFlags = os.constants.dlopen.RTLD_NOW | os.constants.dlopen.RTLD_GLOBAL;
+var foo = require('foo');
+process.dlopenFlags = null;
+```
```

Comment:
Not sure if a property is the right approach. I'd prefer a function (e.g. `setdlopenFlags()`/`getdlopenFlags()` that would include type checking on the value passed.